index.yaml

```diff
@@ -412,9 +412,9 @@ entries:
       catalog.cattle.io/ui-component: longhorn
     apiVersion: v1
     appVersion: v1.1.1
-    created: "2021-06-24T21:41:35.652793-07:00"
+    created: "2021-08-04T15:23:53.922038-07:00"
     description: Longhorn is a distributed block storage system for Kubernetes.
-    digest: bf329bb6f2429e47489851d7355e474c72b590a0123b90fb3ea6eaed5d6d449b
+    digest: e256933b6743544cf832f496c44d391043fe37e6461ac1693cf3c999e0e73378
```

Comment:
Some weird digest changes in there that are unrelated to PR

Reply:
Ohh, it just flipped them, didn't change them.

Reply:
https://github.com/rancher/charts-build-scripts/issues/54 should address this since we should be producing a sorted index.yaml so that it remains consistent on each make charts call